src/views/default/panels/log/detail.php

```diff
@@ -136,7 +136,11 @@
                     $message .= Html::ul($data['trace'], [
                         'class' => 'trace',
                         'item' => static function ($trace) use ($panel) {
-                            return '<li>' . $panel->getTraceLine($trace) . '</li>';
+                            if (isset($trace['file'])) {
+                                return '<li>' . $panel->getTraceLine($trace) . '</li>';
```

Comment:
I would prefer to fix it in the `getTraceLine()` since it looks like it's not protected from invalid data.

Reply:
thanks, i do it.